### ISSUE TITLE
Running time error

### ISSUE THREAD

**Issue by jacklenox** [dularion/streama#452]
I firstly want to thank you for how awesome this is @dularion. I'm delighted with the app and hope I can contribute a bit over time.

I've noticed a strange error where running times appear to be exaggerated by exactly an hour. E.g. I have added a movie which has a running time of 2 hours and 14 minutes, but Streama shows 03:14:56 in the play bar. I've noticed that what happens is that the film time runs as expected for the first hour, but at the mark of the first hour (00:59:59) it skips to being at 2 hours (02:00:00).

This doesn't appear to have happened to all films in my collection, but a significant number of them.

It's a minor issue – now I know it doesn't really cause problems. But an interesting one nonetheless!

**Comment by Jeronimo95** [comment]
That's quite an odd issue. Streama should get this info from the files. Could you run FFprobe on the problem file and post the output?

**Comment by Jeronimo95** [comment]
Also what browser are you using and have you tried on others?

**Comment by jacklenox** [comment]
Thanks @Jeronimo95. Yeah, it's weird. So it appears to be happening in Firefox, but not Chrome. I'm running FF 56.0 (64-bit) for Ubuntu.

I should also make clear, it doesn't actually affect playback. When the timer skips from 00:59:59 to 02:00:00, the playback continues to be smooth. 2 just becomes the new 1. ¯\_(ツ)_/¯

It's happening for all video files that I have.

**Comment by Jeronimo95** [comment]
I just did a quick test with Firefox 56.0.1 on Arch Linux and it seems to be showing the correct running time, but the file I was using was only 21 mins long (I don't have full access to my media files where I am right now). Does it only happen with longer files?

**Comment by jacklenox** [comment]
Yep sorry, I should have been clearer about that. I only see this happen on files that are longer than an hour. Up to an hour, everything looks as expected. I'll try to do a screen record `gif` to show exactly what I'm seeing.

**Comment by Jeronimo95** [comment]
Yeah it might be good to see a gif or something. I just tried with a file that was 1 hour 25 min and it was reporting the time correctly in Firefox 56.0.1 on Arch.

**Comment by dularion** [comment]
Does this only happen with files over 2h? with files over 1h i do not have that issue, times like 1:44:23 appear correctly
